app/views/details/updateAddress.scala.html

```diff
@@ -19,27 +19,35 @@
 @import controllers.manageDetails.UpdateDetailsVM
 @import config.ApplicationConfig
 
+@this(mainLayout: views.html.mainLayout, govukButton: GovukButton)
+
 @(model: UpdateDetailsVM)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@main_template(
-    Messages("updateAddress.title"),
-    topBackLink = true,
-    bottomBackLink = true,
-    backLinkUrl = Some(controllers.manageDetails.routes.ViewDetails.show().url)
+@mainLayout(
+    pageTitle = Messages("updateAddress.title"),
+    mainClass = Some("govuk-grid-column-two-thirds"),
```

Comment:
`govuk-grid-column-two-thirds` is the default css class in mainLayout, so this is not needed